spec/system/provider_interface/provider_manages_application_interviews_spec.rb

```diff
@@ -26,14 +26,18 @@
 
     when_i_visit_that_application_in_the_provider_interface
     and_i_click_set_up_an_interview
-    and_i_fill_out_the_interview_form(days_in_future: 1, time: '12pm')
+    and_i_fill_out_the_interview_form(days_in_future: 0, time: '12:30pm')
```

Comment:
I think it would be clearer to use a future day as it's not easy to understand at a glance that the created interview is actually in the future.

Consider extracting this line in a step with a more descriptive name.

Reply:
I've reverted it to how it was and put more logic in the "When the interview has happened" method.